.evergreen/config.yml

```diff
@@ -668,6 +679,18 @@ axes:
         variables:
            STORAGE_ENGINE: "inmemory"
 
+  - id: libmongoc-version
+    display_name: libmongoc version
+    values:
+      - id: "1.15-latest"
+        display_name: "1.15 latest"
+        variables:
+          LIBMONGOC_VERSION: "r1.15"
+      - id: "master"
+        display_name: "Upcoming release (1.16)"
```

Comment:
Would it make sense to leave this as "(master)" so this never has to be updated? I assume after a libmongoc release is made, we'll need to create a new entry in the `libmongoc-version` axis either way so it's a question of whether we also want to update this parenthetical while doing so.

Reply:
We need to update the pipeline when we switch to a new version of libmongoc, but this can become outdated even when we don't make a change (e.g. once 1.16 is released, as master will be 1.17-dev then). `(master)` will work better.